code/lib/client-logger/src/index.ts

```diff
@@ -1,6 +1,6 @@
-import global from 'global';
+import { global } from '@storybook/global';
 
-const { LOGLEVEL, console } = global;
+const { LOGLEVEL } = global;
 
 type LogLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'silent';
```

Comment:
```suggestion
type LogLevel = typeof LOGLEVEL;
```

Reply:
Ow I see what you're doing here.. I'm afraid this will likely break the TS bundling, because the `typings.d.ts` loading feature isn't automatic.

We'd have to add a tripple-lash directive to make that work. Perhaps we can do the reverse though? We should be able to import the type of  `LogLevel` in the `typings.d.ts` file? That way we have a single source of truth.